### DISCUSSION TITLE
Persistant ZK

### DISCUSSION THREAD

**Discussion by verkaro** [zk-org/zk#293] [Ideas]
Hi, editing and searching markdown I tend to use this bash loop to keep ZK persistant after closing a file.

```
#!/bin/bash
while true
do
  zk edit -i
  sleep 1
done
```

It's a kluge admitedly, but works. My question is if there is any plans to include a ZK mode that makes it persistant? It really would be handy and faster than the second of sleep I've included above so I can Ctrl-C out of the loop.

Cheers

**Comment by mickael-menu** [comment]
Thanks for chiming in. That's stretching a bit the scope of `zk`, which is not meant to be a full-fledged editor/UI. Another issue is that we don't really know when the user closes the editor, if it doesn't block the calling process.

Usually people would open notes [through their editors](https://github.com/mickael-menu/zk-nvim) for persistent sessions, and use `zk edit -i` for a quick modification.

**Reply by verkaro** [reply]
Thank you for taking the time to explain the reasoning and scope. Its a great project and staying within the parameters of intended purpose while developing ZK shows the quality of thought involved. In other words I'm good with that. Cheers.
